composer.json

```diff
@@ -48,31 +66,42 @@
         }
     },
     "extra": {
-        "class": "Acquia\\Blt\\Composer\\Plugin",
         "composer-exit-on-patch-failure": true,
+        "class": "Acquia\\Blt\\Composer\\Plugin",
         "branch-alias": {
-            "dev-master": "9.x-dev"
+            "dev-master": "9.2.x-dev"
```

Comment:
If this does get put into a 10.0.x branch and released as 10.0.0, this will need to be updated.